x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/index.tsx

```diff
@@ -236,22 +243,31 @@ export function Detail() {
 
     return (entries(PanelDisplayNames)
       .filter(([panelId]) => {
-        return (
-          panelId !== 'policies' || packageInfoData?.response.status === InstallStatus.installed
-        );
+        // Don't show `Policies` tab if package is not installed
+        if (panelId === 'policies' && packageInstallStatus !== InstallStatus.installed) {
+          return false;
+        }
+
+        // Don't show `custom` tab if a custom component is not registered
+        if (panelId === 'custom' && !showCustomTab) {
```

Comment:
Q. Should the custom tab be visible to packages that are not installed?

I think thats ok, since the custom tab is meant to show integration specific links/info.

Reply:
I agree, I don't think we should show the tab unless the integration is installed